WhaleSpotting/react-app/src/styles/ShowResultMessage.scss

```diff
@@ -7,6 +7,14 @@
     margin-top: -20px;
 }
 
+.red-background {
+    background-color: indianred;
```

Comment:
We already have a red colour in constants

Reply:
Changed to reject button constant